packages/tools/devtools/devtools-view/src/components/OpLatencyView.tsx

```diff
@@ -44,6 +51,105 @@ const useStyles = makeStyles({
 export function OpLatencyView(): React.ReactElement {
 	const styles = useStyles();
 
+	const messageRelay = useMessageRelay();
+	const [durationOutboundBatchingData, setDurationOutboundBatchingData] =
+		React.useState<GraphDataSet>({
+			graphType: "line",
+			schema: {
+				displayName: "Duration Outbound",
+				uuid: "durationOutbound",
+				xAxisDataKey: "outboundtimestamp",
+				yAxisDataKey: "durationOutboundBatching",
```

Comment:
Nit: I think using `timestamp` and `duration` for all of them is ok. They're all conceptually "duration VS time" charts.